vividus-tests/src/main/resources/story/integration/TableTransformers.story

```diff
@@ -85,3 +85,12 @@ Then `${table[0].name}` is not equal to `${table[0].name}`
 When I initialize story variable `table_resolved` with values:
 /data/with-eagerly-transformer.table
 Then `${table_resolved[0].name}` is equal to `${table_resolved[0].name}`
+
+Scenario: Verify RESOLVING_SELF_REFERENCES_EAGERLY transformer
+Then `<column1>` is equal to `A`
+Then `<column2>` is equal to `A`
+Then `<column3>` is equal to `A`
+Examples:
+{transformer=RESOLVING_SELF_REFERENCES_EAGERLY}
+|column1|column2  |column3  |
```

Comment:
Please cover different cases, like in unit tests? 
asda< column1 >asdasd<column2> etc

Reply:
|column1|column2  |column3  |column 4|
|A      |<column3>|<column1>|